Mixpanel/MPNotification~iphonelandscape.storyboard

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="10116" systemVersion="15E65" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
-        <deployment identifier="iOS"/>
```

Comment:
Why do you want to move this from the storyboard as well? Even if we load this storyboard programmatically, I think it's worth to keep this on the storyboard.